developers/README.md

```diff
@@ -2,28 +2,23 @@
 
 ## Table of Contents
 - [Development Environment](#development-environment)
-  - [Local Environment](#local-environment)
-  - [Virtual Machine](#virtual-machine)
+  - [Virtual Machines](#virtual-machines)
   - [Docker](#docker)
 - [Contributing](#contributing)
 - [IRC](#irc)
 
 ## Development Environment
-Before hacking on BonnyCI, we'll need to set up a development environment. There are three basic ways to do so:
-  1. [On our local machine](#local-environment)
-  2. [In a virtual machine](#virtual-machine)
-  3. [In a Docker container](#docker)
+Before hacking on BonnyCI, we'll need to set up a development environment. There are two basic ways to do so:
+  2. [In virtual machines](#virtual-machines)
+  3. [In Docker containers](#docker)
 
-### Local Environment
+### Virtual Machines
 Supported development platforms:
   - [Ubuntu](dev-environment/ubuntu.md)
```

Comment:
I'm curious as to why the development environment documentation is here rather than in the Development Environment section above. Seems like the content of these two sections has been swapped.